### ISSUE TITLE
Variables `metrics_server_image_tag` and `metrics_server_image` are not in use

### ISSUE THREAD

**Issue by netflash** [cookielab/terraform-kubernetes-metrics-server#4]
Even tho these are optional variables, they are not in use at all.

**Comment by swors** [comment]
I also found the bug. Fix it?

**Comment by Vrtak-CZ** [comment]
> I also found the bug. Fix it?

Can you tell us what kind of bug?

**Comment by swors** [comment]
I wanted to set a user-defined image like it:
```
module "metrics_server" {
  source               = "cookielab/metrics-server/kubernetes"
version              = "0.11.0"
  metrics_server_image = "xxxxxxxxxxxxx"
}
```
but the module still used the built-in image.
I found this code snippet in main.tf:
```         
 image = "k8s.gcr.io/metrics-server-amd64:v0.3.6"
```
I think It should be modified to
```
image = var.metrics_server_image
```
Right?

**Comment by Vrtak-CZ** [comment]
fixed and released as 0.11.1
